src/ai/defaultai.cc

```diff
@@ -102,7 +102,7 @@ DefaultAI::DefaultAI(Widelands::Game& ggame, Widelands::PlayerNumber const pid,
      player_(nullptr),
      tribe_(nullptr),
      attackers_count_(0),
-     next_ai_think_(0),
+     next_ai_think_(1),
```

Comment:
Fix confirmed and working, code LGTM. This line needs a comment why this must not be 0 though or it might be changed again in the future by accident.